Makefile

```
@@ -17,6 +17,9 @@ OS_ARCH ?= $(shell uname -m)
 # Operating system type
 OS_TYPE ?= $(shell uname)
 
+# Suite timeout must be exported so it can be overwritten by set-env-vars.sh
+export SUITE_TIMEOUT = 30s
```

I think the comment about `export` allowing overwrite is incorrect, export in `Makefile` for make variables is useful for [recursive `Makefiles`](https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html), subshells and subcommands but here it doesn't do much

imho we should stick to [conditional assignment](https://www.gnu.org/software/make/manual/html_node/Conditional-Assignment.html) here like all the other make variables above and allow caller to define this, just default here in case it's not defined at all
```suggestion
SUITE_TIMEOUT ?= 30s
```

---

@wozniakjan please run a command test. It will not work without export.

---

It will not work if `SUITE_TIMEOUT` is not provided in the `set-env-vars` and as I understand we should be able to run test without the script.

---

ah, I see, I would still suggest at least using [conditional assignment](https://www.gnu.org/software/make/manual/html_node/Conditional-Assignment.html)
```suggestion
export SUITE_TIMEOUT ?= 30s
```

---

I will try to find some workaround to get rid of that ugly export.

---

it will work ok without the `export` if you use make variable expansion `$()` instead of explicit environment variable expansion `$${}` here
https://github.com/kyma-project/btp-manager/blob/a665e90575e866554186742aff2c29b6752bdb3c/Makefile#L90

https://www.gnu.org/software/make/manual/html_node/Reference.html